feature-003.txt

```diff
@@ -0,0 +1,6 @@
+Feature 3
```

Comment:
Keep up the good work. Finish this course.

Reply:
Thanks, Pradeep!